torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5543,6 +5543,19 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_conversion(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, dtype=dtype, device=device)
+
+    cases = ((),
+             (2, 3))
+    memory_format_options = [None, torch.preserve_format, torch.contiguous_format]
```

Comment:
What about `torch.channels_last`?

```suggestion
    memory_format_options = [None, torch.preserve_format, torch.contiguous_format, torch.channels_last]
```